frontend/src/metabase/collections/components/CollectionHeader/CollectionHeader.styled.tsx

```diff
@@ -30,13 +31,18 @@ interface CollectionHeaderButtonProps {
 export const CollectionHeaderButton = styled(
   Button,
 )<CollectionHeaderButtonProps>`
-  padding: 0.5rem 0.75rem;
-  height: 2.5rem;
+  padding: 0.25rem 0.5rem;
```

Comment:
Use the same value as Maz https://github.com/metabase/metabase/pull/23833/commits/bf2e2d6625657aee776ccdaab61c0d9c04f1b289#diff-5a30b16741703579e402851f93daa3f5b97928cd3d13e0fade912be23c5d2940R39-R41